config.json

```diff
@@ -25,20 +25,11 @@
   "theme": {
     "light": "#f8f9fa",
     "dark": "#343a40",
-    "primary": "#007bff",
```

Comment:
We should add an upgrade guide for the new version if folks are using these colors.

Reply:
Definitely will add some JSON snippets to the README for people wanting to add back these boostrap defaults.